src/test/java/core/basesyntax/service/impl/StrategySelectorTest.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.DaoService;
+import core.basesyntax.db.DaoServiceHashMap;
+import core.basesyntax.model.Operation;
+import core.basesyntax.strategy.SaveStrategy;
+import core.basesyntax.strategy.impl.SaveStrategyBalance;
+import core.basesyntax.strategy.impl.SaveStrategyPurchase;
+import core.basesyntax.strategy.impl.SaveStrategySupply;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class StrategySelectorTest {
+    private static DaoService STORAGE;
+    private static final Map<Operation, SaveStrategy> STRATEGIES_WITHOUT_RETURN =
+            new HashMap<>();
+    private static StrategySelector strategySelector;
+
+    //instead of @BeforeAll
+    static {
+        STORAGE = new DaoServiceHashMap();
+        STRATEGIES_WITHOUT_RETURN.put(Operation.SUPPLY, new SaveStrategySupply(STORAGE));
+        STRATEGIES_WITHOUT_RETURN.put(Operation.PURCHASE, new SaveStrategyPurchase(STORAGE));
+        STRATEGIES_WITHOUT_RETURN.put(Operation.BALANCE, new SaveStrategyBalance(STORAGE));
```

Comment:
you should also write tests for SaveStrategySupply, SaveStrategyPurchase and so on